src/data/option/defs.lean

```diff
@@ -9,6 +9,11 @@ Extra definitions on option.
 namespace option
 variables {α : Type*} {β : Type*}
 
+/-- An elimination principle for `option`. It is a nondependent version of `option.rec_on`. -/
+protected def elim {α β} : option α → β → (α → β) → β
```

Comment:
Do you need `{α β}` here? This way it becomes less clear whether you use `Sort` or `Type` here.

Reply:
I agree it's clearer without it. I just copied it from a file that didn't have the variable declarations.